app/Notifications/NewPost.php

```diff
@@ -74,6 +74,14 @@ public function toArray($notifiable)
             ];
         }
 
+        if ($this->type == 'staffarea') {
+            return [
+                'title' => $this->poster->username.' Has Posted In A Staff Forum Topic',
+                'body'  => $this->poster->username.' has left a new post in Staff Topic '.$this->post->topic->name,
+                'url'   => sprintf('/forums/topics/%s?page=%s#post-%s', $this->post->topic->id, $this->post->getPageNumber(), $this->post->id),
```

Comment:
If the route has a name (which this one does), it's ideal to use the route helper, to avoid hard-coding any portion of the URL, e.g.:

```php
'url'   => sprintf('%s?page=%s#post-%s', route('forum_topic', ['id' => $this->post->topic->id]), $this->post->getPageNumber(), $this->post->id),
```